src/main/webapp/app/exercises/programming/manage/update/update-components/theia/programming-exercise-theia.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="form-group">
+    @if (programmingExercise.allowOnlineIde) {
```

Comment:
if would move this if conditional to the parent component that this component is only displayed when its really needed.

In the parent component sth like

```suggestion
    @if (programmingExercise.allowOnlineIde) {
        <jhi-programming-exercise-theia ... />
    }
```

Reply:
That was the case already, just double-checked it. The second check (as mentioned in your comment) is removed now